### PR TITLE
First Laba

### PR DESCRIPTION
**Pull request by ALHrom**
ДЗ

### CODE REVIEW

File: Grom/ZKAI/Laba1/company/Task10.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.company;
+class Task_10{
+    public double task_10(double n){
```
Comment (secourse2019) [inline Grom/ZKAI/Laba1/company/Task10.java:3]: int n

File: Grom/ZKAI/Laba1/company/Task11.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        return temp;
+    }
+}
+
+
+public class Task11 {
+    public static void main(String[] args) {
+
+        Task_11 b = new Task_11();
+        System.out.println(b.task_11(10));
```
Comment (secourse2019) [inline Grom/ZKAI/Laba1/company/Task11.java:20]: what is 10?

Comment (secourse2019) [reply Grom/ZKAI/Laba1/company/Task11.java:20]: int n = Integerer.valueOf(args[0])

File: Grom/ZKAI/Laba1/company/Task12.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+class Task_12 {
+    public double task_12(double n) {
```
Comment (secourse2019) [inline Grom/ZKAI/Laba1/company/Task12.java:4]: function naming

**Review by secourse2019 — CHANGES_REQUESTED** [review]
ну ты понел

File: Grom/ZKAI/Laba1/company/Task14.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package com.company;
+
+ class Task_14 {
+    public double Task_14(int s){
```
Comment (secourse2019) [inline Grom/ZKAI/Laba1/company/Task14.java:4]: static will be better

File: Grom/ZKAI/Laba1/company/Task14.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        }
+        return s;
+    }
+}
+
+public class Task14{
+    public static void main(String[] args) {
+
+        Task_14 b = new Task_14();
+        System.out.println(b.Task_14(3));
```
Comment (secourse2019) [inline Grom/ZKAI/Laba1/company/Task14.java:17]: with static usage will Task_14.Task_14(3)

**Review by secourse2019 — CHANGES_REQUESTED** [review]
pull code from upstream master and update code

**Review by ALHrom — COMMENTED** [review]
Новое

**Pull request merged**